### DISCUSSION TITLE
Listener UI feedback

### DISCUSSION THREAD

**Discussion by meljoann** [funmusicplace/mirlo#326] [Ideas]
Hey, there are a few things that worked differently to how I expected, when using the site for the first time. This was on Firefox iOS. See what you think:

- [x] I expected clicking on the album artwork to take me to the album, instead I had to click on the text under the artwork, which took me a while to figure out https://github.com/funmusicplace/mirlo/issues/310
- [x] But I loved the play button on the artwork, so handy to be able to listen immediately 
- [ ] I expected clicking on the player at the bottom to bring up a larger version of the player with more options (e.g., could contain full album tracklist, or queue, & wishlist or follow button maybe) https://github.com/funmusicplace/mirlo/issues/332

Also, on my first try the audio was muted. This seems to happen on some sites if my phone is on mute (which is always haha). But not on all sites. On Mirlo, it wasn't muted on the second visit to the site, for some reason. Who knows?

**Comment by LouisLouiseKay** [comment]
FYI Clickable albums are being worked on, like, in mobile view clicking album image will take you to albums but you can click play to play from where you are. This feels important and we've discussed it already but it's great having feedback on it. 

Curious what you think would be best on browser view though, would a "go to album page" under the play button work? The goal being to make it clear there are two different buttons when people hover the album pic.

Similarly, player extended view isn't sorted yet. It's a matter of what we want in the expanded view too!

One thing I'd like is for the expanded view to have purchase options, like a sort of smaller album/artist page where people can follow/subscribe/buy.

Another thing that would be great would be lyrics displayed there.

**Reply by meljoann** [reply]
Amazing! Yeah I guess I expected that because the play button is not part of the album artwork, it would function as a separate button. 

It looks nice to have it on the artwork, but alternatively having the play button underneath could work & be clearer I guess?

**Reply by LouisLouiseKay** [reply]
I think as it is, if the rest of the image is clickable it's straightforward enough (did you except the play button to take you to the album or just when you clicked the picture?). The goal being to free up visual space for readability. Once the image will be clickable we'll see if people still get confused.

Since there's more space in browser view we can try more things.

**Comment by simonv3** [comment]
Linked two tickets in the original for tracking progress on this
